x-pack/plugins/apm/public/components/app/RumDashboard/URLFilter/URLSearch/SelectableUrlList.tsx

```diff
@@ -104,27 +106,30 @@ export function SelectableUrlList({
     }
   };
 
-  // @ts-ignore - not sure, why it's not working
-  useEvent('keydown', onEnterKey, searchRef);
-
   const onInputClick = (e: React.MouseEvent<HTMLInputElement>) => {
     setPopoverIsOpen(true);
+    if (searchRef) {
+      searchRef.focus();
```

Comment:
When the popover does not own focus, focus is placed back on the body. To prevent this focus lost, we explicitly set focus back on the input when clicked.